gmso/formats/gro.py

```diff
@@ -121,6 +121,9 @@ def write_gro(top, filename, precision=3):
         The location and name of file to save to disk.
     precision : int, optional, default=3
```

Comment:
Random question, but if this is written as 
```x=1000; f"{x:.3f}"```
Then the output is 1000.000. Which is 7 significant figures, not 3. Would it be better to define this as the number of decimals? For the LAMMPS writer, I used `n_decimals` as the name of this parameter. Might be good to use something consistenly.

Reply:
Good idea, I have changed it to be `n_decimals` for better agreement between engines